source/index.html.md

```diff
@@ -104,6 +104,157 @@ curl -X DELETE \
 
 You can access Kontist API endpoints by adding `Authorization`: `Bearer AUTH_TOKEN` header to your requests.
 
+## Device Binding
+
+To have access to Kontist API endpoints that require strong customer authentication, you need to pass two-factor authentication (2FA).
+
+To make it seamless for you we provide device binding authentication that leverages digital signature algorithm.
+
+To pass device binding authentication you need to generate a pair of private and public keys by using elliptic curve algorithm  (secp256r1). To create and verify your device you need to pass your public key and then sign OTP from SMS received on your phone.
```

Comment:
nit: there is an extra space between `algorithm` and `(secp256r1)`

Reply:
thanks, fixed